### ISSUE TITLE
How to change app icon

### ISSUE THREAD

**Issue by aadityasivaS** [electron-react-boilerplate/electron-react-boilerplate#2792]
## Summary

Hi How can I change the App Icon I am new to electron but know react very well and I am confused by seeing this many icon files

![image](https://user-images.githubusercontent.com/67770098/110893023-35820080-831b-11eb-8c78-62b675a8f677.png)

Please help

**Comment by balsigergil** [comment]
With electron you can update the icon of the **running** application in the BrowserWindow class. In the boilerplate the icon is set [here](https://github.com/electron-react-boilerplate/electron-react-boilerplate/blob/3cdcbaf873690189a501fd388b5456d229ea17be/src/main.dev.ts#L74). electron-builder is used to create a packaged app and will search for icons in the `assets/icons` folder for the final executable/installer depending of the operating system.

To sum up, `assets/icon.png` is used as icon for the running application and icons in the `assets/icons` folder are used for executables and installers by electron-builder.

**Comment by aadityasivaS** [comment]
Thanks a lot

**Comment by basiclaser** [comment]
any recommendations on generating all the icons from a source file @balsigergil ?

**Comment by tennessine** [comment]
> any recommendations on generating all the icons from a source file @balsigergil ?

https://github.com/onmyway133/IconGenerator/releases

**Comment by praveen-zluri** [comment]
@balsigergil I tried that but the dock icon is still the same also the main app icon that sits in the /Applications is also the old one.

**Comment by avinash6982** [comment]
I am able to change the Dock icon using 
`app.dock.setIcon(image)`
but for me, I couldn't find a way to change the icon that comes up in the notifications(macOS). Any help??!
